panacea/query_client.go

```diff
@@ -495,7 +502,7 @@ func (q verifiedQueryClient) GetOracleUpgradeInfo(ctx context.Context) (*oraclet
 	return &oracleUpgradeInfo, nil
 }
 
-func (q verifiedQueryClient) GetLastBlockHeight(ctx context.Context) (int64, error) {
+func (q *verifiedQueryClient) GetLastBlockHeight(ctx context.Context) (int64, error) {
```

Comment:
I think this function can be removed. This function seems to be replaced by `GetCachedLastBlockHeight()`

Reply:
I have left this function out because some functions may need to get height information that isn't cached.

Reply:
I've modified it to make it better and reusable.

https://github.com/medibloc/panacea-oracle/pull/101/commits/438d139137ae0977b43135fceeb7eea4ce9eb26a